boxsdk/object/group.py

```diff
@@ -14,14 +14,14 @@ class Group(BaseObject):
     _item_type = 'group'
 
     @api_call
-    def get_memberships(self, offset=0, limit=None, fields=None):
+    def get_memberships(self, limit=None, offset=0, fields=None):
```

Comment:
This looks like a breaking change; make sure to document it in the Changelog.  Also, consider giving `offset` a default value of `None` to align with the other methods